app/prices.js

```diff
@@ -56,28 +56,30 @@ exports.init = function (callback) {
                 continue;
             }
 
+            const attributes = getAttributes(pricelist[i].sku);
+
             // Go through pricestf prices
-            for (let j = 0; j < prices.length; j++) {
-                if (pricelist[i].name === prices[j].name) {
+            for (let j = 0; j < prices[attributes.quality][attributes.killstreak].length; j++) {
+                const item = prices[attributes.quality][attributes.killstreak][j];
+
+                if (pricelist[i].sku === item.sku) {
```

Comment:
Check by name instead of sku, it is a cheap fix for a stupid problem but it works.

Reply:
Fixed in [19b635d](https://github.com/Nicklason/tf2-automatic/pull/172/commits/19b635d6892b7362588683b50a11b63121c5c484)